feat: Documentation for Honeycomv v3.3 (#115)
brown-ccv/honeycomb-docs: docs/project_organization/directory_structure.mdx

## node_modules/

:::caution
`node_modules/` is written to when running `npm install` and should never be edited. It is in Honeycomb's `.gitignore` and should never be added to git.
:::

## psiturkit/

The file `psiturk-it` inside `psiturkit/` is a bash script used to instal PsiTurk locally - see [PsiTurk](psiturk#build-instructions) for more information.

:::caution
This folder involves a Honeycomb deployment. The files do not need to be edited.
:::

## public/

The `public` directory contains files that are used as assets in the built app.

- `index.html` is the entry point of the website

>   - Changing `<title>Honeycomb</title>` will update the name you can see in the browser tab!

Author: ```suggestion
  - Changing `<title>Honeycomb</title>` will update the text in the browser tab!

```

## node_modules/

:::caution
`node_modules/` is written to when running `npm install` and should never be edited. It is in Honeycomb's `.gitignore` and should never be added to git.
:::

## psiturkit/

The file `psiturk-it` inside `psiturkit/` is a bash script used to instal PsiTurk locally - see [PsiTurk](psiturk#build-instructions) for more information.

:::caution
This folder involves a Honeycomb deployment. The files do not need to be edited.
:::

## public/

The `public` directory contains files that are used as assets in the built app.

- `index.html` is the entry point of the website
  - Changing `<title>Honeycomb</title>` will update the text in the browser tab!

> 

Author: ```suggestion
  - Changing `<title>Honeycomb</title>` will update the text in the browser tab!
```
